src/iop/colorbalance.c

```diff
@@ -2333,7 +2333,7 @@ void gui_init(dt_iop_module_t *self)
   dt_bauhaus_widget_set_label(g->controls, NULL, _("color control sliders"));
   dt_bauhaus_combobox_add(g->controls, _("HSL"));
   dt_bauhaus_combobox_add(g->controls, _("RGBL"));
-  dt_bauhaus_combobox_add(g->controls, _("both"));
+  dt_bauhaus_combobox_add(g->controls, C_("color control sliders", "both"));
```

Comment:
Do we really need this context so low-level? Having a context name for each module seems wrong to me.

Here we are talking about "options" or a "choice" and we can select "both" of those. In colorreconstruction.c just below we have "none" (same context to me, options or choice).

Reply:
Come to think of it, I have probably chosen to add context to the wrong instance of "both" here - this one is quite fine as it is without any context at all, as the meaning is rather basic here, as opposed to the other instances of "both" where it actually means 'in both directions'. What do you think, would context like "flip/rotation direction" be OK for the other instances?

Reply:
just "direction" maybe or "orientation"?

Reply:
For my purposes right now, it doesn't matter which we use, as long as we do use one. From a general perspective, I'd say "flip direction" is clearer, though (NB: not "flip/rotation direction" as I mistakenly wrote above).

P.S. If another occurrence of "both" gets added somewhere in darktable, chances are the Estonian translation would need it to have its own context anyway (unless it would again be about flipping something), as our word for "both" is inflected in several possible ways depending on, ahem, context :)